docs/src/main/paradox/introduction.md

```diff
@@ -111,7 +111,7 @@ JSON support is possible in `akka-http` by the use of Jackson, an external artif
 for details).
 @@@
 
-The @unidoc[Route] created using the Route DSL is then "bound" to a port to start serving HTTP requests:
+The @scala[@scaladoc[Route](akka.http.scaladsl.server.Route$)]@java[@javadoc[Route](akka.http.javadsl.server.Route)] created using the Route DSL is then "bound" to a port to start serving HTTP requests:
```

Comment:
This is not correct for Scala. In Scala the `Route` we talk about here is: 
`  type Route = RequestContext ⇒ Future[RouteResult]`
as defined on: https://github.com/akka/akka-http/blob/master/akka-http/src/main/scala/akka/http/scaladsl/server/package.scala#L11

Reply:
We can link to that instead I propose, just to the source location

Reply:
Ah, I got it. Thanks for the correction!

> We can link to that instead I propose, just to the source location

I am thinking of linking to the API document: https://doc.akka.io/api/akka-http/current/akka/http/scaladsl/server/index.html#Route=akka.http.scaladsl.server.RequestContext=%3Escala.concurrent.Future[akka.http.scaladsl.server.RouteResult]

Reply:
That sounds okey, please do and then we merge, thanks

Reply:
I went through all of `@unidoc`s . I also found some other small issues. After fixing those and linking to the correct `Route`, I will push soon. :)